src/ensembl/src/content/app/entity-viewer/gene-view/components/default-transcripts-list/DefaultTranscriptsList.tsx

```diff
@@ -32,11 +38,16 @@ import styles from './DefaultTranscriptsList.scss';
 type Props = {
   gene: Gene;
   rulerTicks: TicksAndScale;
+  transcriptscontentUI: EntityViewerGeneViewTranscriptsUI;
```

Comment:
Not sure about the name (what does the word "content" add?), but if we keep it, the `c` should be upper-cased.

Reply:
I missed to remove it. It's done now.